docs/ref/django-admin.txt

```diff
@@ -467,6 +472,19 @@ would search ``<app_label>/fixtures/foo/bar/mydata.json`` for each installed
 application,  ``<dirname>/foo/bar/mydata.json`` for each directory in
 :setting:`FIXTURE_DIRS`, and the literal path ``foo/bar/mydata.json``.
 
+Also, you can use special symbol, ``-``, as fixture name, to take input
+from ``sys.stdin``, for example::
+
+    django-admin loaddata --format=json -
+
+``format`` option is ``required`` in this case, you need to specify format
+of input, registered name of a :ref:`serializer <serialization-formats>`
+(e.g., ``json`` or ``xml``).
+
+You can use it with standard input and output redirections, for example::
+
+    heroku run -r prod django-admin dumpdata --format=json | heroku run -r staging django-admin loaddata --format=json -
```

Comment:
This is already possible without this change with `django-admin loaddata data.json`.

Besides `cat file | command` is an antipattern. `command < file` is more efficient.